src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -698,6 +698,10 @@ void Graph::AllocateWithReuse() {
                 // !! Fallback to individual memory allocation !!
                 // if you like to check infer without reuse just call this function without arguments.
                 edge->allocate(workspace_ptr + offset * alignment);  // alignment in byte
+                if (edge->getMemory().getDesc().getPrecision() == ov::element::string) {
+                    auto str_ptr = reinterpret_cast<std::string *>(edge->getMemory().getData());
+                    std::uninitialized_fill_n(str_ptr, edge->getMemory().getSize() / ov::element::string.size(), std::string());
```

Comment:
I don't see any place where dctor of `std::string` is called in CPU plugin code.
When you created a new memory and initialized with strings, you need to deallocate these strings before main pointer is destroyed.

Reply:
Fixed